application/Chart.yaml

```diff
@@ -6,7 +6,7 @@ type: application
 
 # Helm chart Version
 
-version: 2.2.13
+version: 2.3.0
```

Comment:
add `#minor` in commit message to bump the minor version instead of setting version to `2.3.0`

Reply:
This is done!